cmd/config/init/flags.go

```diff
@@ -44,7 +44,7 @@ func AddFlags(cmd *cobra.Command) error {
 		StringP(
 			FlagNames.DAType,
 			"",
-			"Celestia",
+			"Avail",
```

Comment:
please remove occurances where avail is set as the default da to replace celestia, currently it's not tested/integrated into roller and isn't considered stable

Reply:
Yeah, I will remove those.